chipsec_util.py

```diff
@@ -124,29 +97,15 @@ def parse_args(self):
             logger().DEBUG   = True
         if self.log:
             logger().set_log_file( self.log )
-        if self._cmd_args:
-            self.help_cmd = self._cmd_args[0]
-
-
-    ##################################################################################
-    # Entry point
-    ##################################################################################
-
-
-    def main(self):
-        """
-        Receives and executes the commands
-        """
 
+    def import_cmds(self):
         if self.CHIPSEC_LOADED_AS_EXE:
             import zipfile
             myzip = zipfile.ZipFile("library.zip")
-            cmds = map( self.map_modname_zip, filter(self.f_mod_zip, myzip.namelist()) )
+            cmds = [i.replace('/','.').replace('chipsec.utilcmd.','')[:-4] for i in myzip.namelist() if r'chipsec\/utilcmd\/' in i and i[:-4] == ".pyc" and not i[:2] == '__' ]
```

Comment:
nit: I find more readable   endswith('.pyc')  and startswith('__') instead of the array indexes

Reply:
Sounds good I'll make those changes